example/spec/kinesisTests/KinesisTestTriggerSpec.js

```diff
@@ -88,13 +100,24 @@ describe('The Cloud Notification Mechanism Kinesis workflow', () => {
   beforeAll(async () => {
     try {
       await createOrUseTestStream(streamName);
-      console.log(`\nwaits for active Stream ${streamName}.`);
+      await createOrUseTestStream(cnmResponseStreamName);
+
+      console.log(`\nWaiting for active streams: '${streamName}' and '${cnmResponseStreamName}'.`);
       await waitForActiveStream(streamName);
-      console.log(`Drops record onto  ${streamName}.`);
+      await waitForActiveStream(cnmResponseStreamName);
+
+      console.log(`Dropping record onto  ${streamName}.`);
```

Comment:
can we change to     ` console.log(`Dropping record onto  ${streamName}. recordIdentifier: ${recordIdentifier}.`);` to help with debugging?

Reply:
I can do this when addressing @jennyhliu comment